core/cli/readme.md

```diff
@@ -0,0 +1,11 @@
+# dotcom-tool-kit
```

Comment:
**suggestion**: this readme is going to look a bit weird in isolation on npmjs.com. probably better than the current no readme, but. it could link to the toplevel readme or something?